### DISCUSSION TITLE
I cant log in through email

### DISCUSSION THREAD

**Discussion by alexxsac** [vercel/community#3396] [Help] [answered]
### Summary

Ive tried to sign in and update our account through email but im not getting any verification emails to click and log in through. 

Its crucial to get in and update certain projects. Has anyone encountered this and found a way to fix it?

Ive got in contact with support but idk how fast theyll be, figured id ask the community as well

### Steps to Reproduce

I am clicking sign in, put in my email but never get the code or link to click and verify

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with login. If you already opened a case with the `Registration` problem area, then it should already be in the correct queue and will be addressed fairly quickly. If you let me know the case number, then I can check that it's been routed correctly to the folks who specialize in this problem area.

**Comment by mesorastech** [comment]
I have the same situation, any update?

**Comment by mesorastech** [comment]
Hi,
I can't verify my gmail with this link can I get verify email again?
